drivers/serial/uart_nrfx_uarte.c

```diff
@@ -2042,12 +2183,19 @@ static int uarte_nrfx_pm_action(const struct device *dev,
 
 	switch (action) {
 	case PM_DEVICE_ACTION_RESUME:
+	{
 		ret = pinctrl_apply_state(cfg->pcfg, PINCTRL_STATE_DEFAULT);
 		if (ret < 0) {
 			return ret;
 		}
 
 		nrf_uarte_enable(uarte);
+#if UARTE_BAUDRATE_RETENTION_WORKAROUND
+		nrf_uarte_baudrate_t baudrate = COND_CODE_1(CONFIG_UART_USE_RUNTIME_CONFIGURE,
+			(data->nrf_baudrate), (cfg->nrf_baudrate));
+
+		nrf_uarte_baudrate_set(get_uarte_instance(dev), baudrate);
```

Comment:
Why not just:
```suggestion
		nrf_uarte_baudrate_set(get_uarte_instance(dev),
			COND_CODE_1(CONFIG_UART_USE_RUNTIME_CONFIGURE,
				(data->nrf_baudrate), (cfg->nrf_baudrate)));
```
to avoid those extra braces?